packages/commonwealth/client/scripts/views/modals/delete_address_modal.tsx

```diff
@@ -75,7 +75,7 @@ export const DeleteAddressModal = ({
         notifySuccess('Address has been successfully removed.');
       }
     } catch (err) {
-      notifyError('Address was not successfully deleted, please try again.');
+      notifyError(err.response.data.error);
```

Comment:
prob just use a plain text error

Reply:
these message are dynamic like "magic wallet cannot be removed” , "You are the only admin of this community invite someone and make admin".